Library/Homebrew/test/cask/cask_spec.rb

```diff
@@ -337,6 +337,8 @@
       expect(JSON.pretty_generate(h["variations"])).to eq expected_sha256_variations.strip
     end
 
+    # @note The calls to `APIHashable.generating_hash!` and `APIHashable.generated_hash!`
+    #   are not idempotent so they can only be used in one test.
```

Comment:
This comment should get updated. It's more that you can't have multiple tests for the same class.

Reply:
Maybe I should add it to the `APIHashable` file instead?

Reply:
Sorry, this was more a note to self. 😁

Reply:
Addressed in https://github.com/Homebrew/brew/pull/15007/commits/801ee5e4744ba4f046e01d442ffd749af2d163a5